src/code/z_skelanime.c

```diff
@@ -798,245 +795,299 @@ void SkelAnime_InterpFrameTable(s32 limbCount, Vec3s* dst, Vec3s* start, Vec3s*
     }
 }
 
+static u32 sDisabledTransformTaskGroups = 0;
+static u32 sCurAnimTaskGroup;
+
 /**
- * zeroes out the current request count
+ * Resets the AnimTaskQueue.
+ * Zeroing the task count will stop the queue from processing tasks until new ones are added.
  */
-void AnimationContext_Reset(AnimationContext* animationCtx) {
-    animationCtx->animationCount = 0;
+void AnimTaskQueue_Reset(AnimTaskQueue* animTaskQueue) {
+    animTaskQueue->count = 0;
 }
 
 /**
- * Shifts the queue flag to the next queue
+ * Changes `sCurAnimTaskGroup` to the next group number.
+ *
+ * Task groups allow for disabling "transformative" tasks for a defined group.
+ * For more information see `AnimTaskQueue_DisableTransformTasksForGroup`.
+ *
+ * Note that `sCurAnimTaskGroup` is not a whole number that increments, it is handled at the bit-level.
```

Comment:
I'd prefer this comment to go on `sCurAnimTaskGroup` instead (something like that this is a bitset where exactly one bit it set for the current group)

Reply:
If I was learning about this system for the first time I personally would first look at functions that interface with the system, rather than the piece of data itself. Maybe its worth commenting on both idk

Reply:
"Group numbers are indicated by a single bit that increments by shifting 1 position to the left. See `AnimTaskQueue_SetNextGroup` for more details"

would you be fine with this comment on the data member?

Reply:
Maybe something general about group numbers would be helpful, but for the data members I'd prefer the comment to be more precise about what the data member itself represents, e.g.

```
// A bitset where the `i`th bit is set if the `i`th task group is disabled.
static u32 sDisabledTransformTaskGroups = 0;
// `(1 << i)` where `i` is the current task group.
static u32 sCurAnimTaskGroup;
```

Armed with this knowledge, the reader can guess that `sCurAnimTaskGroup <<= 1` is conceptually incrementing the "current task group", and something like `sCurAnimTaskGroup & sDisabledTransformTaskGroups` is testing whether the "current task group" is "disabled".